src/redux/actions/fetchMovieDescription.js

```diff
@@ -0,0 +1,6 @@
+export const fetchMovieDescription = () => {
+  dispatch({
+    type: "FETCH_MOVIE_DESCRIPTION",
+    payload: movie
+  });
+};

```

Comment:
You don't have dispatch function here, but you have it in `mapDispatchToProps` function. This is second parameter of `connect` HOC from `react-redux`
```
export const fetchMovieDescription = () => ({
    type: "FETCH_MOVIE_DESCRIPTION",
    payload: movie
  });
```
Also it's a bit confusing naming because fetch means that this action should request data but not save it. It would be better to name like `saveMovieDescription`. You can use fetch naming but for setting loading for example